webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -183,6 +239,28 @@ export default {
     }),
   },
   methods: {
+    unBlur() {
+      if (this.checkedBlur) {
+        this.checkedChange()
+      }
+    },
+    checkedChange() {
+      if (this.$el) {
+        this.elem = this.$el.querySelector('img')
+      } else {
+      }
+      if (this.checkedBlur) {
+        this.elem.classList.remove('img-blur-in')
+        this.checkedBlur = false
+        this.form.checkbox = false
+      } else {
+        if (this.elem != null) {
+          this.elem.classList.add('img-blur-in')
+        }
+        this.checkedBlur = true
+        this.form.checkbox = true
+      }
+    },
```

Comment:
I think we can improve this! there is an empty else after `line 250`... 
there are nested `if` inside `else` clauses, which could be refactored to use `else if` statements, no?
Also, it's a bit complicated to follow what's going on here... what are you trying to achieve in `checkedChange`?
I see it is called when `unBlur` is called, and when the component is `mounted`